src/pages/home/report/ReportDropUI.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {View} from 'react-native';
+import {Portal} from '@gorhom/portal';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import DragAndDropIcon from '../../../../assets/images/drag-and-drop.svg';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const ReportDropUI = props => (
```

Comment:
I think it would be good to merge `DragAndDrop` in this component.

Reply:
What do you mean by this? I like the utility of having a reusable `DragAndDrop` component.

Reply:
I don't consider this component as a utility. It is tied to a single use. Although, there is no issue keeping it separate.

Reply:
I definitely prefer the separation of concerns established with the current design